packages/device-registry/src/test/DeviceFacade.test.ts

```diff
@@ -113,17 +104,14 @@ describe('Device Facade', () => {
                 smartMeter: { address: deviceSmartmeter },
                 owner: { address: deviceOwnerAddress },
                 lastSmartMeterReadWh: 0,
-                status: DeviceStatus.Active,
-                usageType: Device.UsageType.Producing,
-                lastSmartMeterReadFileHash: 'lastSmartMeterReadFileHash',
-                propertiesDocumentHash: null,
-                url: null
+                lastSmartMeterReadFileHash: 'lastSmartMeterReadFileHash'
             };
 
-            const devicePropsOffChain: ProducingDevice.IOffChainProperties = {
+            const devicePropsOffChain: IDevice = {
+                status: DeviceStatus.Active,
                 operationalSince: 0,
                 capacityInW: 10,
-                country: 'Thailand',
+                country: 221,
```

Comment:
What does 221 means?

Reply:
That's the ID of the country in the Countries array. This was just for tests, I will replace it with `Countries.find(c => c.name === 'Thailand').id`